src/encoded/static/components/auditmatrix.js

```diff
@@ -228,7 +228,7 @@ class AuditMatrix extends React.Component {
             const type = pluralize(queryStringType.toLocaleLowerCase());
 
             return (
-                <div>
+                <React.Fragment>
```

Comment:
Could the previous `<div>` have just been deleted? It only surrounds one div. Not sure if there was a CSS style that relied on it or not.